packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -20,12 +20,12 @@
 
             <lightning-button-group>
                 <template if:true={canShowMore}>
-                    <lightning-button variant="base" label="More" title="Show more" icon-name="utility:arrowdown"
-                                      class="slds-m-left_x-small" onclick={showMore}></lightning-button>
+                    <lightning-button variant="base" label="More" title={labels.showMore} icon-name="utility:arrowdown"
+                                      class="slds-m-left_x-small" onclick={showMore} aria-label={showMoreFacetValuesLabel}></lightning-button>
                 </template>
                 <template if:true={canShowLess}>
-                    <lightning-button variant="base" label="Less" title="Show less" icon-name="utility:arrowup"
-                                      class="slds-m-left_x-small" onclick={showLess}></lightning-button>
+                    <lightning-button variant="base" label="Less" title={labels.showLess} icon-name="utility:arrowup"
+                                      class="slds-m-left_x-small" onclick={showLess} aria-label={showLessFacetValuesLabel}></lightning-button>
```

Comment:
Those aria made me think. Do we have the capability to test automaticly for accessiblity ?
I know that is a requirement for atomic iirc

Reply:
We currently don't have it implemented but there's a [jira for that](https://coveord.atlassian.net/browse/SFINT-3969) and yes there are tests we can run to check for accessibility. Atleast to a certain degree.